### PR TITLE
Version 1.1.3 + re-rendered feedstock

### PR DESCRIPTION
**Pull request by easz**
respond to issue #24 and #18

Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by easz** [comment]
To make CI work I fixed some outdated packages (e.g. torch -> pytorch) and also manually added `pytorch` channel in ci files. (actually it should be done with `conda_build_config.yaml` and `conda-smithy` or `[at]conda-forge-admin please rerender` later). 

However the Windows CI is still failing. I will try try to re-render feedstock to see if it will also work on Windows CI.

**Comment by easz** [comment]
hi @bw4sz @henrykironde 
please have a look if the change is OK.

and I hope that you did not get too many spam emails due to the PR.

File: recipe/conda_build_config.yaml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+channel_sources:
+  - pytorch,conda-forge,defaults
```
Comment (isuruf) [inline recipe/conda_build_config.yaml]: We don't allow channels outside of `conda-forge, defaults` to be in `channel_sources` in `conda-forge`.

Comment (easz) [reply recipe/conda_build_config.yaml]: OK, I will try w/o pytorch channel

Comment (easz) [reply recipe/conda_build_config.yaml]: @isuruf   but what is the purpose? of
~~~
Specify the feedstock channel and label: Optionally, you can specify source channels and choose a channel to upload to in recipe/conda_build_config.yaml.

channel_sources:
  - mysourcechannel1,mysourcechannel2,conda-forge,defaults
channel_targets:
  - target_channel target_label
~~~ 
from 

https://github.com/conda-forge/conda-smithy#making-a-new-feedstock

for none-conda-forge?

Comment (isuruf) [reply recipe/conda_build_config.yaml]: There are other github orgs (other than conda-forge) using conda-smithy to generate feedstocks. Only conda-forge feedstocks disallow channels outside conda-forge.

**Comment by easz** [comment]
basically it comes back to the initial problem with Windows CI.

there is no win-64 `torchvision` from `conda-forge, defaults` channels and adding `pytorch` channel in conda-forge's feedstocks is not allowed.

need another solution :p

**Comment by bw4sz** [comment]
thanks for the thoughts. This is exactly where I got stuck last time. There is a long thread in the torch conda-forge build on this.

https://twitter.com/jeremyphoward/status/1178351261608861701?lang=en

I do not see a way forward right now. @conda-forge/torchvision what is the current recommended way of using torch/torchvision in a conda-forge project?

**Comment by hmaarrfk** [comment]
unfortunately, there is no recommended way. tests aren't passing on the CIs so it is making us worried to release a new package of torch vision.if you can help debug the package, it would be greatly appreciated

**Comment by easz** [comment]
@bw4sz the easiest "workaround" at the moment would be inherently a build without Windows CI....   :(

**Comment by hmaarrfk** [comment]
Why is this not a noarch python package?

**Comment by hmaarrfk** [comment]
See suggestions here:
https://github.com/conda-forge/deepforest-feedstock/pull/28

I won't be updating that pull request more than it currently is.

**Comment by easz** [comment]
see #28

**Pull request closed without merging**